app/policies/encryptable_policy.rb

```diff
@@ -31,4 +33,11 @@ def destroy?
   def team
     @record.team
   end
+
+  private
+
+  def transferring_encryptable?
```

Comment:
maybe rename this method to receiving_encryptable, as I understand this method allows encryptables to be created when receiving one, and not for transfering (sending), but I might just misunderstand the context

Reply:
No it is the method which checks if the encryptable is transferring when sharing a new one, because when you share an encryptable, then the sender can't be teammember if the receiver's personal team.